src/endpoint/s3/s3_rest.js

```diff
@@ -252,7 +252,8 @@ async function authorize_request_policy(req) {
     const account = req.object_sdk.requesting_account;
     const is_nc_deployment = Boolean(req.object_sdk.nsfs_config_root);
     const account_identifier_name = is_nc_deployment ? account.name.unwrap() : account.email.unwrap();
-    const account_identifier_id = is_nc_deployment ? account._id : undefined;
+    // Both NSFS NC and containerized will validate bucket policy against acccount id.
```

Comment:
In NC, the account identifier by ID served both accounts and users, as we didn't have the ARN yet.

Reply:
@shirady started a thread in [slack](https://noobaa.slack.com/archives/C09U45P0XKP/p1764159890494479) to discuss this